File: Data Structures & Algorithms/merge-strings-alternately/submission-0.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int size = min(word1.length(),word2.length());
        {
            string s="";
            for(int i=0;i<size;i++)
            {
                s=s+word1[i]+word2[i];
            }
            if(word1.length()>word2.length())
            {
                for(int i=size;i<word1.length();i++)
                {
                    s+=word1[i];
                }
            }
            else
            {
                for(int i=size;i<word2.length();i++)
                {
                    s+=word2[i];
                }
            }
            return s;
        }
    }
};

File: Data Structures & Algorithms/minimum-size-subarray-sum/submission-0.java
class Solution {
    public int minSubArrayLen(int target, int[] nums) {
        int sum = 0;
        int mini = Integer.MAX_VALUE;
        int left = 0;
        for(int right = 0; right<nums.length; right++)
        {
            sum += nums[right];
            while(sum>=target && left<nums.length)
            {
                mini = Math.min(mini, right-left+1);
                sum -= nums[left];
                left++;
            }
        }
        if(mini==Integer.MAX_VALUE) return 0;
        return mini;
    }
}